schema/revert/shipping/views.sql

```diff
@@ -740,6 +740,16 @@ comment on view shipping.return_results_v2 is
     'Version 2 of view of barcodes and presence/absence results for return of results on website';
 
 
+revoke all
+    on shipping.hcov19_observation_v1
+  from "hcov19-visibility";
+drop view shipping.hcov19_observation_v1;
+
+revoke all
+    on shipping.fhir_encounter_details_v2
+  from "incidence-modeler";
+drop view shipping.fhir_encounter_details_v2;
```

Comment:
Why the revokes here before the drop?

Reply:
🤷‍♀ I have no idea how I got it in my head that was necessary...